tests/UI/campaigns/functional/BO/13_shopParameters/06_trafficAndSeo/01_seoAndUrls/05_seoOptions/01_displayAttributesInProductMetaTitle.js

```diff
@@ -0,0 +1,102 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const seoAndUrlsPage = require('@pages/BO/shopParameters/trafficAndSeo/seoAndUrls');
+const foHomePage = require('@pages/FO/home');
+const productPage = require('@pages/FO/product');
+
+// Import data
+const {Products} = require('@data/demo/products');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shopParameters_TrafficAndSeo_seoAndUrls_displayAttributesInProductMetaTitle';
+
+
+let browserContext;
+let page;
+
+describe('Enable/Disable display attributes in product meta title', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to \'Shop parameters > SEO and Urls\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToSeoAndUrlsPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.shopParametersParentLink,
+      dashboardPage.trafficAndSeoLink,
+    );
+
+    await seoAndUrlsPage.closeSfToolBar(page);
+
+    const pageTitle = await seoAndUrlsPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(seoAndUrlsPage.pageTitle);
+  });
+
+  const tests = [
+    {args: {action: 'enable', enable: true, metaTitle: `${Products.demo_1.name} Size S Color White`}},
```

Comment:
IDK if the combination type should not be in the description of the product. Maybe add it as a child ?

Reply:
Done :+1: thx :)